Library/Homebrew/tap.rb

```diff
@@ -31,7 +31,7 @@ def self.fetch(*args)
     repo = repo.strip_prefix "homebrew-"
 
     if user == "Homebrew" && repo == "homebrew" ||
-        user == "Linuxbrew" && repo == "linuxbrew"
+        repo == "linuxbrew"
```

Comment:
You need to update https://github.com/Linuxbrew/linuxbrew/blob/master/Library/Homebrew/cmd/test-bot.rb#L44
to
```ruby
if (tap = ENV["TRAVIS_REPO_SLUG"]) && (tap =~ %r{^([\w-]+)/(homebrew-)?([\w-]+)$})
```